### PR TITLE
feat: retest: Scan Rule ID table values no longer displayed with commas

### PR DESCRIPTION
**Pull request by kingthorin**
Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

File: addOns/retest/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## Unreleased
 ### Changed
 - Update minimum ZAP version to 2.11.1.
+- Plugin Id values are no longer displayed with commas.
```
Comment (thc202) [inline addOns/retest/CHANGELOG.md:10]: `Scan Rule ID`, I know it's not what's currently shown but we should correct that (if not in this PR in other).

Comment (kingthorin) [reply addOns/retest/CHANGELOG.md:10]: I could tweak that here too.

File: addOns/retest/src/main/resources/org/zaproxy/addon/retest/resources/Messages.properties [hunk 2]
```diff
@@ -4,7 +4,7 @@ retest.plan.name = Retest Plan
 
 retest.dialog.title = Retest
 retest.dialog.table.header.status = Status
-retest.dialog.table.header.pluginid = Plugin Id
+retest.dialog.table.header.pluginid = Scan Rule ID
```
Comment (thc202) [inline addOns/retest/src/main/resources/org/zaproxy/addon/retest/resources/Messages.properties:7]: Sorry for pushing more work, could you rename the resource keys as well?

**Comment by kingthorin** [comment]
Tweaked.

File: addOns/retest/CHANGELOG.md (hunk 1 above)
Comment (thc202) [reply addOns/retest/CHANGELOG.md:10]: This one was not changed, on purpose?

Comment (kingthorin) [reply addOns/retest/CHANGELOG.md:10]: Oh no, I just looked past it somehow. I'll do this one too but it won't happen till the morning for me.

**Comment by kingthorin** [comment]
CHANGELOG now tweaked as well.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**